src/lib/github/updateExistingIssues.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * API function to add comments to existing bot created issues.
+ */
+
+import { Context } from "./types";
+import { Issue } from "./addIssuesToRepo";
+import { getBasicRepoProps } from "./utils";
+
+type Issues = Issue[];
+
+export default async function updateExistingIssues (context: Context, newIssue: Issue): Promise<Issues> {
```

Comment:
Return type should be Promise<void> since the final statement in this function is `forEach` whose return type is void.